requirements.txt

```diff
@@ -70,9 +70,10 @@ geoip2==2.9.0
 gevent==1.4.0
 gipc==1.0.1
 greenlet==0.4.15
-gunicorn==19.9.0
-hiredis==1.0.0
-html5lib==0.9999999
+grpcio==1.30.0
+gunicorn==20.0.4
+hashids==1.2.0
+html5lib==1.0.1
```

Comment:
I don't know if these libraries are compatible with python 2.
We should only import what we know we'll need (`redis-py-cluster` in this case)